bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/model/Evaluation.kt

```diff
@@ -0,0 +1,15 @@
+package io.bucketeer.sdk.android.internal.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Evaluation(
+  val id: String,
+  val feature_id: String,
+  val feature_version: Int,
+  val user_id: String,
```

Comment:
I kept snake_case field name for model classes to keep it simple.
BKTEvaluation and other user-facing classes will be camel-cased.